### ISSUE TITLE
Whitelist / Oplist not listing players

### ISSUE THREAD

**Issue by rickselby** [msmhq/msm#274]
msm &lt;server> wl add &lt;player> (etc) works, but msm &lt;server> wl list (etc) doesn't show the list of players.

Certainly affects whitelists and operators, not tried on banlist.

**Comment by minnesotags** [comment]
Same behavior with 1.8.1 on Debian.

**Comment by loredan** [comment]
Yeah, msm is reading white-list.txt file on ```msm <server> wl list``` command. And looks like msm uses it to add players when server is offline. An easy fix

**Comment by stevenroose** [comment]
Is this project actually even maintained...

**Comment by renderorange** [comment]
@stevenroose The msm repo has an open commit, so anyone who has had a previously accepted merge request can accept merge requests.  Perhaps it's risky, but it puts the pressure on the community itself.  Maybe you :)

There are test which run on merge requests that could use some love and updating, but I would be hesitant to rely solely on those, even if every function were covered.  There's also a fair amount of information in previous issues on how to bump versions, as well as the versioning scheme itself.

Admittedly, I was helping keep things moving for a while, but my available time has gone thin for now.  I suspect it's going to be like that for most people who contribute.
